### PR TITLE
Fix code style

### PR DESCRIPTION
**Pull request by dluc**

### CODE REVIEW

File: samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs [hunk 1]
```diff
@@ -20,6 +18,7 @@
 using SemanticKernel.Service.Model;
 using SemanticKernel.Service.Skills;
 using SemanticKernel.Service.Storage;
+using Directory = System.IO.Directory;
```
Comment (stephentoub) [inline samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs:21]: Why do this rather than `using System.IO;`?

Comment (dluc) [reply samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs:21]: less code/readability (btw, I don't do this manually, it's an automation code style fix in R#)

Comment (stephentoub) [reply samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs:21]: `using System.IO;` is less code than `using Directory = System.IO.Directory;` :)

Comment (dluc) [reply samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs:21]: sorry, I see what you mean, there's an ambiguous ref, the same Directory class in MsGraph lib

Comment (stephentoub) [reply samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs:21]: > another Directory class in MsGraph lib

😦

**Review by hathind-ms — APPROVED** [review]

**Pull request merged**